src/ElectricPokemon.java

```diff
@@ -0,0 +1,48 @@
+package src;
+
+public class ElectricPokemon extends Pokemon{
+    private Integer height;
+    private String food;
+
+    public ElectricPokemon(String name, Integer level, Integer height, String food) {
+        super(name, level);
+        this.height = height;
+        this.food = food;
+    }
+
+    @Override
```

Comment:
Je hebt de overrides goed toegepast, maar de implementatie is nu bij elke subklasse bijna hetzelfde, waardoor je het ook met simpele parameters had kunnen oplossen.